packages/menu/test/menu.test.ts

```diff
@@ -76,6 +78,35 @@ describe('Menu', () => {
         await elementUpdated(el);
         expect(document.activeElement === el, 'self not focused, 2').to.be
             .false;
+        expect(focusinSpy.callCount).to.equal(0);
+    });
+    it('renders w/ all [disabled] menu items', async () => {
+        const focusinSpy = spy();
+        const el = await fixture<Menu>(
+            html`
+                <sp-menu tabindex="0" @focusin=${() => focusinSpy()}>
```

Comment:
@adixon-adobe `keydown` listening is only bound _after_ a `focusin` event, this ensure that we don't do any work around this as `focusin` is prevented when the `sp-menu` element is `focus()`ed and there are no children that can accept focus as well as when a child that cannot accept focus is `focus()`ed.